### DISCUSSION TITLE
Data streams are described by the "Stream" class

### DISCUSSION THREAD

**Discussion by galenlynch** [AllenNeuralDynamics/aind-data-schema#811] [Q&A]
Brought up by @alakunina

Data streams (a group of streams) are described by the class `Stream`, which is confusing.

https://github.com/AllenNeuralDynamics/aind-data-schema/blob/c593512d06d7e76a25cb94735d1989ce226dc5e1/src/aind_data_schema/core/session.py#L230-L231

**Comment by alakunina** [comment]
Brought up specifically because a group of Neuropixels will have slightly differing start and stop times, though perhaps the difference is small enough that it doesn't need to be described in the metadata file?

**Comment by alakunina** [comment]
Though things like the cameras can have their start/stop times be off by seconds from the ephys data, as we generally start the recording first, then start the camera acquisition, to ensure all frame times are captured as events.

**Comment by saskiad** [comment]
Can you describe the issue a bit more for me? Is the issue that different devices have different times and should be separate Streams? Or is it something else? I would have the recording define the start/end and not specify exactly when cameras start since that will be reflected in the synchronization, correct? But if the cameras start/end outside of the recording - eg if the recording can stop but the cameras continue - then perhaps they need to be a distinct stream.
